app/components/table-responsive/README.md

```diff
@@ -65,7 +65,7 @@ import ProgressBar from "./app/components/progress-bar";
           <ProgressBar modifier="green" value="1" max="5" />
         </Td>
         <Td column="Nettopreis">
-          <Amount>2.575,00 €</Amount>
+          <Amount currency="EUR" locale="en-US">{2575.00}</Amount>
```

Comment:
Sorry to barge in here, but why is this not `<Amount … value={2575} />`? Strikes me as odd/unintuitive to use child elements for this, as evidenced by the error. (This might be a topic for discussion elsewhere, some other time.)

Reply:
Joy and I just discussed this, we will keep it like this for now – but we can discuss it, when the conference is over 😉

Reply:
counterproposal: We live-stream our discussion to the room, thus proving that component-based design is a mistake.